source/content/Character/Classes/Cleric/Miracles/Level 1/Command.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Command
+tags:
+  - level_1
+  - cleric
+---
+**Cleric Level 1**
+**Duration**: 1 Round
+**Range**: 10'
+
+When a Cleric casts this spell, he can give the subject a single word command, which it obeys to the best of its ability. The single word must make sense as a single command, such as approach, drop, fall, flee, halt, surrender, sleep, etc. Although a target could be instructed to die, this will only make the target collapse into a comatose state for a single Round. Note that the caster must be able to speak the language of the target. Any intended target that has more than 5 levels or Hit Dice, or an Intelligence of over 12 is entitled to a saving throw versus Magic. This spell is ineffective against undead.
```

Comment:
"speak the language of the target" -> "target must comprehend the speech of the caster" or smth.
languages spell is applicable here, so the caster might not know the language, but still communicate